doc/nrf/gs_installing.rst

```diff
@@ -31,40 +37,53 @@ Install the required tools
 
 The installation process is different depending on your operating system.
 
+.. note::
+      You will be asked to reboot after installing some of the tools.
+      You can skip these notifications and reboot only once after you complete the installation of all tools.
+
 .. tabs::
 
    .. group-tab:: Windows
 
       The recommended way for installing the required tools on Windows is to use `Chocolatey`_, a package manager for Windows.
       Chocolatey installs the tools so that you can use them from a Windows command prompt.
 
-      To install the required tools, complete the following steps:
+      To install the required tools, complete the following steps for Windows from the Install dependencies section in Zephyr's :ref:`zephyr:getting_started`, then return to this guide.
 
-      .. ncs-include:: getting_started/index.rst
```

Comment:
This was because of the last line that mentions using the regular user mode for the command window. But we also mentioned this last line can be avoided with a different include directive, so that's what I'm going to do here after forgetting to do it straight away.

Reply:
Fixed.